packages/react/package.json

```diff
@@ -42,7 +42,7 @@
 		"@dataconnect/default-connector": "workspace:*"
 	},
 	"peerDependencies": {
-		"firebase": "^11",
+		"firebase": "^11.3.0",
```

Comment:
@maneesht could it be that we are now supporting firebase versions `11.3.x` only?